### PR TITLE
Update beerus.ms

### PR DESCRIPTION
**Pull request by Bcrulez**

### CODE REVIEW

File: CommandHelper/LocalPackages/commands/buffs/beerus.ms [hunk 1]
```diff
@@ -21,13 +21,15 @@
...
+                "/touchkill"
             ),
             'buffs': array(
                 "/effect ". @plr ." 5 9999 20 true",
                 "/effect ". @plr ." 10 9999 1 true",
                 "/effect ". @plr ." 11 9999 3 true",
                 "/effect ". @plr ." 13 9999 1 true",
                 "/effect ". @plr ." 21 9999 19 true",
-                "/give ". @plr ." minecraft:iron_boots 1 8:30 4:30 name:&4S&ep&4e&ce&ed&8s&0t&9e&1r_&4Boots"                
+                "/give ". @plr ." minecraft:iron_boots 1 8:30 4:30 name:&4S&ep&4e&ce&ed&8s&0t&9e&1r_&4Boots"
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/beerus.ms]: needs comma on line 32

File: CommandHelper/LocalPackages/commands/buffs/beerus.ms [hunk 2]
```diff
@@ -21,13 +21,15 @@
                 "/fly",
                 "/carryplayer <player>",
                 "/dismountplayer"
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/beerus.ms]: needs comma on line 23

File: CommandHelper/LocalPackages/commands/buffs/beerus.ms [hunk 3]
```diff
@@ -1,4 +1,4 @@
-@plr = player()
+,@plr = player()
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/beerus.ms]: remove the comma here

Comment (Bcrulez) [inline CommandHelper/LocalPackages/commands/buffs/beerus.ms]: ok

**Comment by Bcrulez** [comment]
where might I find permission? Sorry its been awhile since I first started

**Comment by Bcrulez** [comment]
nvm I found it

**Pull request merged**